api/utils/keys/privatekey_test.go

```diff
@@ -18,76 +18,38 @@ package keys
 
 import (
 	"bytes"
+	"crypto"
 	"crypto/ecdsa"
 	"crypto/ed25519"
+	"crypto/elliptic"
+	"crypto/rand"
 	"crypto/rsa"
 	"crypto/tls"
 	"encoding/pem"
 	"testing"
 
-	"github.com/gravitational/trace"
 	"github.com/stretchr/testify/require"
 )
 
-// TestParsePrivateKey tests that ParsePrivateKey successfully parses private key PEM.
-func TestParsePrivateKey(t *testing.T) {
```

Comment:
Should we keep the coverage for the parsing error scenarios?

Reply:
I don't think it's really necessary, which is why I opted into compressing the 2 tests into 1.